docs/_docs/maintaining/merging-a-pull-request.md

```diff
@@ -38,8 +38,9 @@ The categories match the H3's in the history/changelog file, and they are:
 1. Major Enhancements (`+major`) – major updates or breaking changes to the code which necessitate a major version bump (v3 ~> v4)
 2. Minor Enhancements (`+minor`) – minor updates (feature, enhancement) which necessitate a minor version bump (v3.1 ~> v3.2)
 3. Bug Fixes (`+bug`) – corrections to code which do not change or add functionality, which necessitate a patch version bump (v3.1.0 ~> v3.1.1)
-4. Site Enhancements (`+site`) – changes to the source of https://jekyllrb.com, found in `site/`
+4. Site Enhancements (`+site` or `+doc`) – changes to the source or the documentation found in `docs/` and published on https://jekyllrb.com, 
```

Comment:
I think `+doc` routes to `Documentation`, not `Site Enhancements`. https://github.com/parkr/auto-reply/blob/e5c55feb1811aad9f2426fbe49891c50a3376e8f/chlog/merge_and_label.go#L54-L59